recipes/jemalloc/all/conanfile.py

```diff
@@ -1,11 +1,14 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, MSBuild, tools
-from conans.errors import ConanInvalidConfiguration
+from conan import ConanFile
+from conans import AutoToolsBuildEnvironment, MSBuild
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.scm import Version
+from conans import tools as tools_legacy
+from conan.tools.files import apply_conandata_patches, get, rename, replace_in_file
 import os
 import shutil
 import string
 
-required_conan_version = ">=1.33.0"
-
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```
1.51.3 is more than enough here, only if you are using `export_conandata_patches` 1.52.0 will be required.